README.md

```diff
@@ -37,6 +37,15 @@ Run `python3 -m twine upload --repository testpypi dist/*`
 `--repository testpypi` can be removed to push to the prod PyPI instance.
 Also `dist/*` can be a bit more precise to upload a specific version of the binaries
 
+# Update the package version
+
+To update the version use the hatch command:
+
+```
+hatch version fix
```

Comment:
Would `fix` be the default command we'd use?
I guess we're most likely to use the aliases `major` and `minor`?
maybe we just can show the generic command `hatch version "X.Y.Z"`?